app/scripts/controllers/otherResources.js

```diff
@@ -49,8 +49,16 @@ angular.module('openshiftConsole')
       .get($routeParams.project)
       .then(_.spread(function(project, context) {
         $scope.kinds = _.filter($scope.kinds, function(kind){
-          var resource = APIService.kindToResource(kind.kind);
-          return AuthorizationService.canI("list", kind.group ? kind.group + "/" + resource : resource, $scope.projectName);
+          var resource = {
```

Comment:
`resource.resource` is a little awkward, but I don't have a great suggestion. `var resourceAndGroup = ...` maybe?